### DISCUSSION TITLE
Can't use ampersand (&) in math formulas (inside dolar signs)

### DISCUSSION THREAD

**Discussion by stereobooster** [community/community#46788] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Formula: `$\&$` renders `Misplaced &`

Example: $\&$

Was reported in this comment as well https://github.com/community/community/discussions/41087#discussioncomment-4363871

Related https://github.com/nschloe/github-math-bugs

**Comment by stereobooster** [comment, marked as answer]
Posted and immediately found solution 🤦‍♀️ 

`$\\&$` -> $\\&$

**Reply by nschloe** [reply]
Yeah, this is a [known bug](https://github.com/nschloe/github-math-bugs#backslashes-in--math). In $-math, you need to escape backslashes _if_ they come before non-alphanumeric characters. Things like `\Gamma` are fine; also, backtick math is not affected.

**Comment by nschloe** [comment]
The new backtick-delimited inline math syntax allows using regular LaTeX syntax:

```markdown
$`\&`$
```

> $`\&`$
